src/components/Camera/index.js

```diff
@@ -52,28 +54,27 @@ type CameraPureType = {
   ...CameraCommonType,
   onFallbackClick: void => void,
   faceCaptureClick: void => void,
-  countDownRef: React.Ref<typeof Countdown>,
   webcamRef: React.Ref<typeof Webcam>,
 }
 
 // Specify just a camera height (no width) because on safari if you specify both
 // height and width you will hit an OverconstrainedError if the camera does not
 // support the precise resolution.
 const CameraPure = ({method, autoCapture, title, subTitle, onUploadFallback, onFallbackClick,
-  onUserMedia, faceCaptureClick, countDownRef, webcamRef, isFullScreen, onWebcamError, i18n}: CameraPureType) => (
+  onUserMedia, faceCaptureClick, webcamRef, isFullScreen, onWebcamError, i18n}: CameraPureType) => (
```

Comment:
More of a suggestion than anything required - there seems to a fair bit of plumbing for the `isFullScreen` prop, but isn't this something that could be calculated/retrieved in a higher order component? Then you could just wrap all the components that need it with the HoC, and they can respond as necessary, rather than plumbing the prop? This, naturally, depends on where the isFullScreen originates of course and how much further data plumbing is needed for it :)